### PR TITLE
add 562 _() scenario Close the Gaps & standardization of translation contexts of 6 scenarios

### PR DESCRIPTION
**Pull request by muerteFR**
7 GMButton
76 scienceStationDescription

### CODE REVIEW

**Comment by muerteFR** [comment]
@daid this PR is over. I will update the translations of the descriptions after that. This PR is already very big.

File: scripts/scenario_50_gaps.lua [hunk 1]
```diff
@@ -40,11 +42,11 @@ function init()
 	psb["Nautilus"] = 2
 	psb["Hathcock"] = 4
 	-- square grid deployment
-	fleetPosDelta1x = {0,1,0,-1, 0,1,-1, 1,-1,2,0,-2, 0,2,-2, 2,-2,2, 2,-2,-2,1,-1, 1,-1}
-	fleetPosDelta1y = {0,0,1, 0,-1,1,-1,-1, 1,0,2, 0,-2,2,-2,-2, 2,1,-1, 1,-1,2, 2,-2,-2}
+	fleetPosDelta1x = {0,1,0,-1, 0,1,-1, 1,-1,2,0,-2, 0,2,-2, 2,-2,2, 2,-2,-2,1,-1, 1,-1,0, 0,3,-3,1, 1,3,-3,-1,-1, 3,-3,2, 2,3,-3,-2,-2, 3,-3,3, 3,-3,-3,4,0,-4, 0,4,-4, 4,-4,-4,-4,-4,-4,-4,-4,4, 4,4, 4,4, 4, 1,-1, 2,-2, 3,-3,1,-1,2,-2,3,-3,5,-5,0, 0,5, 5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,5, 5,5, 5,5, 5,5, 5, 1,-1, 2,-2, 3,-3, 4,-4,1,-1,2,-2,3,-3,4,-4}
```
Comment (daid) [inline scripts/scenario_50_gaps.lua:45]: There are more changes here then just translation?

Comment (muerteFR) [reply scripts/scenario_50_gaps.lua:45]: This is a bug corrected by @Xansta after I told him about it (on the Quitoxic difficulty) and updated myself as I was already working on this scenario for translations. I say it in the Commit of 15112021 https://github.com/daid/EmptyEpsilon/pull/1617/commits/3035895212e45cd4dc3e3d7786b3d67ef1e92e27

Comment (muerteFR) [reply scripts/scenario_50_gaps.lua:45]: So @daid  if you can validate it I could continue to clean up my French translations :-) and the other translators as well.

**Pull request merged**